diktat-common/src/main/kotlin/org/cqfn/diktat/common/config/rules/RulesConfigReader.kt

```diff
@@ -148,3 +148,10 @@ fun List<RulesConfig>.isRuleEnabled(rule: Rule): Boolean {
     val ruleMatched = getRuleConfig(rule)
     return ruleMatched?.enabled ?: true
 }
+
+/**
+ * Get disable chapters from configuration
+ *
+ * @return parameter in [DIKTAT_COMMON]
+ */
+fun List<RulesConfig>.disabledChapters() = getCommonConfig()?.configuration?.get("disableChapters")
```

Comment:
How it can be lazy? And I have method, that returns `Chapters`, but here, in `RulesConfigReader` it's better to split logic, I think.

Reply:
I think logic of parsing strings shouldn't be in `Warnings.isRuleFromActiveChapter`. And you call that method for every `warn` invocation. Not sure about `lazy` now, but you could hide this check inside `configs.isRuleEnabled`, which is called in both `warn` and `fix`.

Reply:
Since `diktat-common` is unaware of `Chapters`, let's not merge methods, but call both `isRuleFromActiveChapter` and `isRuleEnabled` in `warn` and `fix`.